projects/optic/src/__tests__/integration/diff.test.ts

```diff
@@ -12,7 +12,10 @@ jest.setTimeout(30000);
 
 describe('diff', () => {
   test('two files, no repo or config', async () => {
-    const workspace = await setupWorkspace('diff/files-no-repo');
+    const workspace = await setupWorkspace('diff/files-no-repo', {
+      repo: true,
```

Comment:
Seems like maybe the test name here should be updated if we're including a repo on the test.

Reply:
Ah this should actually be `repo: false` - updated test